src/librustc_typeck/diagnostics.rs

```diff
@@ -12,6 +12,16 @@
 
 register_long_diagnostics! {
 
+E0062: r##"
+This error indicates that during an attempt to build a struct or struct-like
+enum variant, one of the fields was specified more than once.
```

Comment:
Perhaps this could elaborate in saying that each field must be specified exactly one time?